test/unit/rules/no-duplicate-id-test.js

```diff
@@ -8,6 +8,9 @@ generateRuleTests({
   good: [
     // Unique sibling TextNode IDs
     '<div id="id-00"></div><div id="id-01"></div>',
+    '<div id={{unique-id}}></div><div id={{unique-id}}></div>',
+    '<div id="{{unique-id}}"></div><div id="{{unique-id}}"></div>',
+    "<div id='{{unique-id}}'></div><div id='{{unique-id}}'></div>",
```

Comment:
Should we include these two test cases? It's a bit confusing to me since these are just strings that look like the helper.

Reply:
it's not a strings, it's concat statement, and we check what identificators correctly handled for ` and " inside concats.

<img width="1152" alt="image" src="https://user-images.githubusercontent.com/1360552/213922336-76847ea8-dd30-43af-bd8e-f4cd988a3f99.png">